src/rules/noUnusedVariableRule.ts

```diff
@@ -15,6 +15,7 @@
 */
 
 var OPTION_CHECK_PARAMETERS = "check-parameters";
+var OPTION_LEADING_UNDERSCORE = "leading-underscore";
```

Comment:
let's rename the parameter to "ignore-leading-underscore" or actually just "ignore-underscore" to make it less verbose. also please update the README.